benchmarking/mexican-standoff/mexican-3-players-2-lives.prism

```diff
@@ -38,31 +38,31 @@ module shooter2 = shooter1 [health1 = health2, health2 = health1, p1wait = p2wai
-[p1wait,p2wait,p3wait] true -> true;
-[p1wait,p2shoot3,p3shoot1] true -> (health1' = max(health1-1, 0)) & (health3' = max(health3-1, 0));
-[p1wait,p2shoot3,p3shoot2] true -> (health2' = max(health2-1, 0)) & (health3' = max(health3-1, 0));
-[p1wait,p2shoot3,p3wait] true -> (health3' = max(health3-1, 0));
-[p1shoot2,p2shoot1,p3shoot1] true -> (health1' = max(health1-2, 0)) & (health2' = max(health2-1, 0));
-[p1shoot2,p2shoot1,p3shoot2] true -> (health1' = max(health1-1, 0)) & (health2' = max(health2-2, 0));
-[p1shoot2,p2shoot1,p3wait] true -> (health1' = max(health1-1, 0)) & (health2' = max(health2-1, 0));
-[p1shoot2,p2wait,p3shoot1] true -> (health1' = max(health1-1, 0)) & (health2' = max(health2-1, 0));
-[p1shoot2,p2wait,p3shoot2] true -> (health2' = max(health2-2, 0));
-[p1shoot2,p2wait,p3wait] true -> (health2' = max(health2-1, 0));
-[p1shoot2,p2shoot3,p3shoot1] true -> (health1' = max(health1-1, 0)) & (health2' = max(health2-1, 0)) & (health3' = max(health3-1, 0));
-[p1shoot2,p2shoot3,p3shoot2] true -> (health2' = max(health2-2, 0)) & (health3' = max(health3-1, 0));
-[p1shoot2,p2shoot3,p3wait] true -> (health2' = max(health2-1, 0)) & (health3' = max(health3-1, 0));
-[p1shoot3,p2shoot1,p3shoot1] true -> (health1' = max(health1-2, 0)) & (health3' = max(health3-1, 0));
-[p1shoot3,p2shoot1,p3shoot2] true -> (health1' = max(health1-1, 0)) & (health2' = max(health2-1, 0)) & (health3' = max(health3-1, 0));
-[p1shoot3,p2shoot1,p3wait] true -> (health1' = max(health1-1, 0)) & (health3' = max(health3-1, 0));
-[p1shoot3,p2wait,p3shoot1] true -> (health1' = max(health1-1, 0)) & (health3' = max(health3-1, 0));
-[p1shoot3,p2wait,p3shoot2] true -> (health2' = max(health2-1, 0)) & (health3' = max(health3-1, 0));
-[p1shoot3,p2wait,p3wait] true -> (health3' = max(health3-1, 0));
-[p1shoot3,p2shoot3,p3shoot1] true -> (health1' = max(health1-1, 0)) & (health3' = max(health3-2, 0));
-[p1shoot3,p2shoot3,p3shoot2] true -> (health2' = max(health2-1, 0)) & (health3' = max(health3-2, 0));
-[p1shoot3,p2shoot3,p3wait] true -> (health3' = max(health3-2, 0));
+  [p1wait,p2shoot1,p3shoot1] true -> (health1' = (health1-2));
+  [p1wait,p2shoot1,p3shoot2] true -> (health1' = (health1-1)) & (health2' = (health2-1));
+  [p1wait,p2shoot1,p3wait] true -> (health1' = (health1-1));
+  [p1wait,p2wait,p3shoot1] true -> (health1' = (health1-1));
+  [p1wait,p2wait,p3shoot2] true -> (health2' = (health2-1));
+  [p1wait,p2wait,p3wait] true -> true;
+  [p1wait,p2shoot3,p3shoot1] true -> (health1' = (health1-1)) & (health3' = (health3-1));
+  [p1wait,p2shoot3,p3shoot2] true -> (health2' = (health2-1)) & (health3' = (health3-1));
+  [p1wait,p2shoot3,p3wait] true -> (health3' = (health3-1));
+  [p1shoot2,p2shoot1,p3shoot1] true -> (health1' = (health1-2)) & (health2' = (health2-1));
+  [p1shoot2,p2shoot1,p3shoot2] true -> (health1' = (health1-1)) & (health2' = (health2-2));
+  [p1shoot2,p2shoot1,p3wait] true -> (health1' = (health1-1)) & (health2' = (health2-1));
```

Comment:
Why were the `max`s removed here @larsbpf ?

Reply:
I added a range to health instead from [0..max_health]